src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php

```diff
@@ -227,6 +259,138 @@ public function iShouldNotBeLoggedIn()
     }
 
     /**
+     * @When I register with email :email and password :password
+     */
+    public function iRegisterWithEmailAndPassword($email, $password)
+    {
+        $this->registerPage->open();
+        $this->registerPage->specifyEmail($email);
+        $this->registerPage->specifyPassword($password);
+        $this->registerPage->verifyPassword($password);
+        $this->registerPage->specifyFirstName('Carrot');
+        $this->registerPage->specifyLastName('Ironfoundersson');
+        $this->registerPage->register();
+    }
+
+    /**
+     * @Then my account should be verified
+     */
+    public function myAccountShouldBeVerified()
+    {
+        $user = $this->sharedStorage->get('user');
+        $this->securityService->logIn($user->getEmail());
+
+        Assert::true(
+            $this->dashboardPage->isVerified(),
+            'You should be verified.'
+        );
+    }
+
+    /**
+     * @When I use it to verify
+     */
+    public function iUseItToVerify()
+    {
+        $user = $this->sharedStorage->get('user');
+
+        $this->verificationPage->verifyAccount($user->getEmail(), $user->getEmailVerificationToken());
+    }
+
+    /**
+     * @When I resend the verification email
+     */
+    public function iResendVerificationEmail()
+    {
+        $this->dashboardPage->open();
+        $this->dashboardPage->pressResendVerificationEmail();
+    }
+
+    /**
+     * @When I use the verification link from the first email to verify
+     */
+    public function iUseVerificationLinkFromFirstEmailToVerify()
+    {
+        $user = $this->sharedStorage->get('user');
+        $token = $this->sharedStorage->get('verification_token');
+
+        $this->verificationPage->verifyAccount($user->getEmail(), $token);
+    }
+
+    /**
+     * @When I (try to) verify using email :email and token :token
```

Comment:
`When I verify using email :email and token` would not be matched with this, because of two spaces required between I and verify. `When I (try to )verify using email :email and token :token`